R/frontline_dm.R

```diff
@@ -23,6 +23,19 @@ col.date <- function(df){
   return(df)
 }
 
+#' @title Data management for reporting column
+#' @description Filter to only "reporting"
+#' @param df
+#' @return data.frame
+col.reporting <- function(df){
+  
+  df <- df %>% 
+    dplyr::mutate(reporting = ifelse(reporting_status == "Reporting", T, F)) %>% 
+    dplyr::filter(reporting)
+  
+  return(df)
+}
```

Comment:
add reporting column. they're using this (i think instead of keeping date col updated) to signify providers that reported in the current quarter of data